### ISSUE TITLE
MQTT 2 reload

### ISSUE THREAD

**Issue by sasbibic** [eclipse/smarthome#6698]
Version 2.4 MQTT: When things config file updates I get this message into my log:
2018-12-19 09:31:34.520 [hingStatusInfoChangedEvent] - 'mqtt:topic:mymqtt' changed from ONLINE to OFFLINE (COMMUNICATION_ERROR): java.lang.Exception: No MQTT client

MQTT stops working, restarting bundle in console:
231 │ Active   │  80 │ 0.10.0.oh240           │ Eclipse SmartHome MQTT Transport Bundle
resolves problem.

**Comment by shaguarger** [comment]
Upgraded today to openHAB 2.4.0 and encountering the same problem as decsribed :/

**Comment by Str8t** [comment]
Same here, tested on 2.4.0 and the latest 2.5.0 snapshot, running in docker.

**Comment by davidgraeff** [comment]
> latest 2.5.0 snapshot

This issue is fixed in OH 2.5, at least in the underlying ESH.

**Comment by FlorianSW** [comment]
So, the bug will stay in the current stable version? :/

**Comment by davidgraeff** [comment]
Yes, I'm afraid so. openHAB doesn't have bugfix releases, because lack of manpower.
